src/Uart/Uart.c

```diff
@@ -53,6 +53,38 @@ getAddressBase(Uart_Id id)
     }
 }
 
+static inline bool
+setBaudRate(const Uart* const uart, const Uart_BaudRate baud)
+{
+    uint32_t baudValue = 0;
+    switch(baud) {
+        case 300:
+        case 600:
+        case 1200:
+        case 1800:
+        case 2400:
+        case 4800:
+        case 9600:
+        case 19200:
+        case 28800:
+        case 38400:
+        case 57600:
+        case 76800:
+        case 115200:
+            baudValue = (uint32_t) baud;
+            break;
+        default:
+            return false;
+    }
+
+    uint32_t freq = rtems_clock_get_ticks_per_second();
+    uint16_t scaler = freq / (baudValue * 8) - 1;
```

Comment:
Add comment referencing GR712RC manual and the relevant formula

Reply:
Done